lib/primer/classify.rb

```diff
@@ -133,80 +136,94 @@ def validated_class_names(classes)
       #
       # styles_hash - A hash with utility keys that mimic the interface used by https://github.com/primer/components
       #
-      # Returns a string of Primer CSS class names to be added to an HTML class attribute
+      # Returns a string of Primer CSS class names and style attributes to be added to an HTML tag.
       #
       # Example usage:
-      # extract_hash({ mt: 4, py: 2 }) => "mt-4 py-2"
-      def extract_hash(styles_hash)
-        memo = { classes: [], styles: +"" }
-        styles_hash.each do |key, value|
-          next unless VALID_KEYS.include?(key)
+      # extract_css_attrs({ mt: 4, py: 2 }) => "mt-4 py-2"
+      def extract_css_attrs(styles_hash)
+        classes = []
+        styles = +""
 
+        styles_hash.each do |key, value|
           if value.is_a?(Array)
             raise ArgumentError, "#{key} does not support responsive values" unless RESPONSIVE_KEYS.include?(key) || Primer::Classify::Utilities.supported_key?(key)
 
             value.each_with_index do |val, index|
-              Primer::Classify::Cache.read(memo, key, val, BREAKPOINTS[index]) || extract_value(memo, key, val, BREAKPOINTS[index])
+              extract_css_attrs_into(classes, styles, key, val, BREAKPOINTS[index])
             end
           else
-            Primer::Classify::Cache.read(memo, key, value, BREAKPOINTS[0]) || extract_value(memo, key, value, BREAKPOINTS[0])
+            extract_css_attrs_into(classes, styles, key, value, BREAKPOINTS[0])
           end
         end
 
-        memo[:classes] = memo[:classes].join(" ")
+        {
+          class: classes.join(" "),
+          style: styles
+        }
+      end
+
+      def extract_css_attrs_into(classes, styles, key, val, breakpoint)
```

Comment:
reading this method I asked myself "into what?"

Reply:
Ah yeah I know what you mean. I was trying to convey that `classes` and `styles` would be mutated as a side-effect of calling the method. Is there a better name that conveys the same idea or should I just get rid of the "into"?

Reply:
I think removing the `into` would be fine